### ISSUE TITLE
[Frontend] Resolve accessibility violation in Dark Mode Theme Engine

### ISSUE THREAD

**Issue by emdevelopa** [emdevelopa/Stellar_Payment_API#1358]
Description
This task involves UX enhancement for the Dark Mode Theme Engine module.
The goal is to resolve accessibility violation in Dark Mode Theme Engine to improve the platform's styling, user interactions, and overall accessibility.

Requirements and context

- Must be secure, tested, and documented
- Adhere to the Drips Wave design and UI/UX standards
- Ensure compatibility across desktop and mobile browsers
- Specifically focused on frontend UX enhancement and responsiveness

Suggested execution

Fork the repo and create a branch
git checkout -b feature/fe-resolve-accessibility-violation-in-dark-mode-theme-engine
Implement changes

Review existing component in Dark Mode Theme Engine
Apply changes: Resolve accessibility violation in Dark Mode Theme Engine
Use clean CSS or tailwind variables for styling
Maintain state transitions and visual feedback
Test and commit

Test mobile responsiveness and interactive states
Check accessibility (a11y) using standard audits
Include screenshots or gifs in the PR
Example commit message
feat(frontend): resolve accessibility violation in Dark Mode Theme Engine

Guidelines

Complexity: Medium (150 points)
Assignment required before starting
PR description must include: Closes #[issue_id]

**Comment by otobongdev** [comment]
@otobongdev has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> Hello maintainer, i can fix this issue


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/db92d7ca-1ba9-4367-ad91-d336f6c0bc50) or assign @otobongdev to this issue.

**Comment by nonsobethel0-dev** [comment]
@nonsobethel0-dev has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> I have view the issue and I can definitely fix the issue


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/db92d7ca-1ba9-4367-ad91-d336f6c0bc50) or assign @nonsobethel0-dev to this issue.
